backend/fleet_management/migrations/0011_add_default_groups.py

```diff
@@ -0,0 +1,32 @@
+
+from django.db import migrations, connections
+from django.contrib.auth.models import Group
+from ..models import User
+
+
+from ..constants import Groups
+
+
+def create_groups(apps, schema_editor):
+    with connections['default'].cursor() as cursor:
+        cursor.execute(f"INSERT INTO auth_group (\"name\") values ('{Groups.Passenger.name}')")
```

Comment:
Could you use python instead of raw SQL insert? That's bit ugly in my opinion.
Even more, you can use Django's `RunSQL` instead of `cursor` but that would still be not needed.

You're using `Group` model few lines below anyway, so why not here?

```suggestion
        Group.objects.create(name=Groups.Passenger.name)
```

Reply:
That sounds good :)

Reply:
Also - thanks for the hint!